scripts/vellum-to-hq

```diff
@@ -107,7 +107,7 @@ def build_vellum(path, branch, test=True):
 
 
 def get_old_vellum_rev(git):
-    commit = str(git('log', '--oneline', '--grep', 'vellum:', '-n', '1'))
+    commit = str(git('log', '--oneline', '--perl-regexp', '--grep', '(?<!Revert ")vellum:', '-n', '1'))
```

Comment:
Carrying on from my previous example, I think this will still return the wrong result.

commits:
* `vellum:a` <- we want this (a)
* `vellum:b` <- we get this (b)
* `Revert vellum:b`